### PR TITLE
Add omronsentech_camera to documentation index for ROS Melodic.

### PR DESCRIPTION
**Pull request by ose-support-ros**
Hi,

I would like my omronsentech_camera to be indexed and documented for ROS Melodic.

Thanks,

OSE ROS Support.

### CODE REVIEW

File: melodic/distribution.yaml [hunk 1]
```diff
@@ -4356,6 +4356,17 @@ repositories:
       url: https://github.com/ros-drivers/omron.git
       version: kinetic-devel
     status: maintained
+  omronsentech_camera:
+    doc:
+      type: git
+      url: https://github.com/ose-support-ros/omronsentech_camera.git
+      version: master
+    source:
+      test_commits: false
```
Comment (clalancette) [inline melodic/distribution.yaml]: Please remove the `test_commits` line here; false is the default, and we only need to add the line if we want to enable it.

**Review by clalancette — CHANGES_REQUESTED** [review]
One minor change needed.

File: melodic/distribution.yaml (hunk 1 above)
Comment (ose-support-ros) [reply melodic/distribution.yaml]: Hi,

The test_commits has been removed.

Thanks,

OSE ROS Support.

Comment (clalancette) [reply melodic/distribution.yaml]: Thanks!

**Review by clalancette — APPROVED** [review]

**Pull request merged**